recipe/meta.yaml

```diff
@@ -11,11 +11,12 @@ package:
 
 source:
   url: https://github.com/dmlc/xgboost/releases/download/v{{ version }}/xgboost-{{ version }}.tar.gz
-  sha256: a84fc7d9846c24659a2ad16788a7eefa9640b19eea9bbc65f30e0a9d53c52453
+  sha256: 90b1b7b770803299b337dd9b9206760d9c16f418403c77acce74b350c6427667
   patches:
     # xgboost patches
-    - patches/0001-Enable-latest-libcxx-on-MacOS.patch  # [osx]
-    - patches/0002-Mark-wheels-as-any-platform-compatible.patch
+    - patches/0001-Enable-latest-libcxx-on-MacOS.patch                # [osx]
+    - patches/0002-Remove-nvidia-nccl-cu12-from-pyproject.toml.patch
+    - patches/0003-Mark-wheels-as-any-platform-compatible.patch
```

Comment:
Looks like this patch no longer applies

Did this go upstream? Wasn't seeing it in the [`2.1.2` to `2.1.3` diff]( https://github.com/dmlc/xgboost/compare/v2.1.2...v2.1.3 ), but maybe I'm missing something

Reply:
No, the patch had to be refreshed